src/Journey.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const nunjucks = require('express-nunjucks');
+
```

Comment:
Yeah, originally I had wanted to this to be separated as well. The downside is that it means we can't handle the common express stuff as well, like template engine, serving assets, security stuff.

:thinking: Actually, looking back through the Divorce frontend it looks like the only parts that couldn't be encapsulated here would be:

```
    app.set('view engine', 'html');
    app.set('views', [
        __dirname + '/app/steps',
        __dirname + '/app/components',
        __dirname + '/app/views',
        __dirname + '/node_modules/govuk_template_jinja/views/layouts/'
    ]);

    nunjucks(app, {
        autoescape: true,
        watch: true,
        noCache: true
    });
```

so actually that's not so bad. Your app.js would be:
```
const app = express();
app.set('view engine', 'html');
app.set('views', [ ... ]);
nunjucks(app);

const journey = Journey(...)
...
```